src/material/datepicker/calendar.ts

```diff
@@ -282,6 +282,12 @@ export class MatCalendar<D> implements AfterContentInit, AfterViewChecked, OnDes
   /** End of the comparison range. */
   @Input() comparisonEnd: D | null;
 
+  /** ARIA Accessible name of the `<input matStartDate/>` */
+  @Input() startDateAccessibleName: string | null;
```

Comment:
Do we need something for the comparison range as well?

Reply:
We would probably need to when we do comparison range. Wasn't planning on addressing comparison range in this PR. Do you think that's something simple that I could add to this PR?